.github/CHANGELOG.md

```diff
@@ -380,6 +380,11 @@
 
 <h3>Improvements</h3>
 
+* Changed the default precision used by the JAX device. It remains unchanged for
+  `jax.config.config.read('jax_enable_x64')==False` and changed to double precision
+  if `jax.config.config.read('jax_enable_x64')==True`.
+  [(#1485)](https://github.com/PennyLaneAI/pennylane/pull/1485)
```

Comment:
```suggestion
* The precision used by `default.qubit.jax` now matches the value of 
  `jax.config.config.read('jax_enable_x64')`, allowing double-precision computation.
  [(#1485)](https://github.com/PennyLaneAI/pennylane/pull/1485)
```
Maybe a bit simpler?

Reply:
Good call. Although the referenced value is a boolean, so very strictly, the precision is only _indicated_ by `jax.config.config.read('jax_enable_x64')` and not _equal_ to it.
Also, I noticed that `jax.config.config.read('jax_enable_x64')` does not work, but
```python
from jax.config import config
config.read('jax_enable_x64')
```
does.

Reply:
suggested something